actions/crib-purge-environment/action.yml

```diff
@@ -0,0 +1,60 @@
+name: crib-purge-environment
+description: |
+  Action to destroy CRIB epehemeral environment.
+  It requires to run crib-deployment-environment beforehand 
+  and depends on the environment setup from a dependent composite action.
+
+inputs:
+  namespace:
+    description: CRIB Namespace that should be destroyed
+    required: true
+  # grafana inputs (optional)
+  metrics-job-name:
+    description: "grafana metrics job name"
+    required: false
+  metrics-id:
+    description:
+      "grafana metrics id, used for continuity of metrics during job name
+      changes  - required if metrics-job-name is passed"
+    required: false
+  gc-host:
+    description: "grafana hostname - required if metrics-job-name is passed"
+    required: false
+  gc-basic-auth:
+    description: "grafana basic auth - required if metrics-job-name is passed"
+    required: false
+  gc-org-id:
+    description:
+      "grafana org/tenant id - required if metrics-job-name is passed"
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Tear down CRIB ephemeral environment
+      if: always()
```

Comment:
that doesn't seem to work unfortunately, probably because we're inside a composite action and entire thing is skipped